packages/next/client/on-demand-entries-client.js

```diff
@@ -4,7 +4,16 @@ import Router from 'next/router'
 import fetch from 'unfetch'
 
 const { hostname, protocol } = location
-const wsProtocol = protocol.includes('https') ? 'wss' : 'ws'
+const isHttps = protocol.includes('https')
+
+//
+// Cannot use `wss` until we can spin up ws server with ssl certs
+//
+const wsProtocol = 'ws'
+//
+// If developing under https, use ws://localhost
+//
+const wsHostname = isHttps ? 'localhost' : hostname
```

Comment:
This will break in cases where the user isn't running on `localhost`

Reply:
I cannot connect from `https://my-domain.com` to `ws://my-domain.com`. However, I've found that browsers allow connections to `ws://localhost` from different hostnames and protocols.